src/data.table.h

```diff
@@ -33,6 +33,9 @@ typedef R_xlen_t RLEN;
 #define SIZEOF(x) sizes[TYPEOF(x)]
 #define TYPEORDER(x) typeorder[x]
 
+#define INTERNAL_ERROR(s) error("%s %s: %s. %s", _("Internal error in"), __func__, s, _("Please report to the data.table issues tracker"))
+#define INTERNAL_ERRORF(f, ...) error("%s %s: %s. %s", _("Internal error in"), __func__, printf(f, __VA_ARGS__), _("Please report to the data.table issues tracker"))
```

Comment:
i wasn't seeing a platform-robust way of accepting 0 arguments to `...` so it seems we need two macros